### ISSUE TITLE
Why is it abandoned?

### ISSUE THREAD

**Issue by gotzmann** [rryqszq4/ngx_python#1]
Just curious, why the deveopment of this module was abandoned? I saw you've made a big progress with PHP module for Nginx.

**Comment by rryqszq4** [comment]
At present I have no more time to develop this module,  and already has a better implementation about https://github.com/arut/nginx-python-module.

Maybe continue this module in the future.

**Comment by gotzmann** [comment]
Yeah, I saw the module of arut but it was abandoned long ago too unfortunately
